### DISCUSSION TITLE
Why Gemini-1.0-pro-latest ? Why not Gemini-l.5-pro-latest ?

### DISCUSSION THREAD

**Discussion by Enlight432** [immersive-translate/immersive-translate#2077] [Help] [answered]
### Summary

![image](https://github.com/user-attachments/assets/ca1cbdd2-9e79-4913-87a4-69d8fa7e726d)

Why is **gemini-1.0-pro-latest** selected by default, but **gemini-l.5-pro-latest** is not selected ?
Or just answering this question which one is better is enough for me, THX

**Comment by theowenyoung** [comment, marked as answer]
Gemini-l.5-pro-latest is better. Gemini-l.0-pro-latest is cheaper.
